rest/serializers.py

```diff
@@ -0,0 +1,94 @@
+"""
+REST serializers for taxonomy models
+"""
+
+from __future__ import unicode_literals
+
+from rest_framework.serializers import (
+    ModelSerializer,
+    HiddenField,
+    CurrentUserDefault,
+    CreateOnlyDefault,
+)
+from rest_framework.generics import get_object_or_404
+
+from taxonomy.models import Vocabulary, Term
+from learningresources.models import Repository
+from .util import LambdaDefault
+
+
+class RepositorySerializer(ModelSerializer):
+    """Serializer for Repository"""
+
+    created_by = HiddenField(
+        default=CreateOnlyDefault(CurrentUserDefault())
+    )
+
+    class Meta:
+        # pylint: disable=missing-docstring
+        model = Repository
+        fields = (
+            'id',
+            'slug',
+            'name',
+            'description',
+            'create_date',
+            'created_by',
+        )
+        read_only_fields = (
+            'id',
+            'slug',
```

Comment:
`create_date` should probably be in `read_only_fields` also; it's populated automatically when the record is first created, and should never be modified.